wolox-books/src/app/screens/book/components/list/list.component.html

```diff
@@ -1,3 +1,11 @@
 <div class="search">
   <input type="text" class="input-search" placeholder="Busc&aacute; por t&iacute;tulo del libro...">
 </div>
+<div *ngFor="let book of books" class="book-container">
+  <div class="book-card" *ngFor="let number of [0,1,2,3,4,5]">
```

Comment:
I did this to show more books, because I am doing the request to apiary and this shows only one book.

Reply:
Why not usign the actual api?

Reply:
doesn't work the heroku app

Reply:
then... use a property in the class. 🤷🏻‍♀️